.github/workflows/snyk-pr.yml

```diff
@@ -0,0 +1,31 @@
+name: snyk-pr
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  whitesource:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request.user.login == 'sfc-gh-snyk-sca-sa' }}
```

Comment:
This user should be added to the CLA exceptions

Reply:
how can I add this user to CLA exceptions?